Broken `docker build` command in Makefile

**What happened**:
```sh
$ make image-local-build
BUILDER=beautiful_jang
make image-build PUSH=
make[1]: Entering directory '/home/elevran/work/go/src/github.com/neuralmagic/gateway-api-inference-extension'
✔ Loaded DEV_VERSION from .version.json: 0.0.3
✔ Loaded PROD_VERSION from .version.json: 0.0.3
✔ Loaded IMAGE_TAG_BASE from .version.json: quay.io/vllm-d/gateway-api-inference-extension-dev
🛠 Final values: DEV_VERSION=0.0.3, PROD_VERSION=0.0.3, IMAGE_TAG_BASE=quay.io/vllm-d/gateway-api-inference-extension-dev
==== Building Docker image quay.io/vllm-d/gateway-api-inference-extension:0.0.1 ====
docker podman build --build-arg TARGETOS= --build-arg TARGETARCH= -t quay.io/vllm-d/gateway-api-inference-extension:0.0.1 .
unknown flag: --build-arg

Usage:  docker [OPTIONS] COMMAND [ARG...]

Run 'docker --help' for more information
make[1]: *** [Makefile:479: image-build] Error 125
```

**What you expected to happen**:
container image builds correctly

**How to reproduce it (as minimally and precisely as possible)**:
Building in Linux VM, only docker is installed. Run `make image-local-build`

**Anything else we need to know?**:
Makefile seems to pick up the wrong value for CONTAINER_TOOL. It is set to `docker podman ...` as can be seen in the output above.

Commenting out line 400 fixes the issue and allows the build to complete:
`#CONTAINER_TOOL := $(shell command -v docker >/dev/null 2>&1 && echo docker || command -v podman >/dev/null 2>&1 && echo podman || echo "")
`

**Environment**:
- Kubernetes version (use `kubectl version`): NA
- Inference extension version (use `git describe --tags --dirty --always`): latest `dev` branch
- Cloud provider or hardware configuration: NA
- Install tools: docker, KIND, etc,
- Others: NA

--- a/Makefile
+++ b/Makefile
@@ -396,11 +396,11 @@ SHELL := /usr/bin/env bash
 PROJECT_NAME ?= gateway-api-inference-extension
 DEV_VERSION ?= 0.0.1
 PROD_VERSION ?= 0.0.0
-IMAGE_TAG_BASE ?= quay.io/vllm-d/$(PROJECT_NAME)
+IMAGE_TAG_BASE ?= quay.io/vllm-d/$(PROJECT_NAME)/epp
 IMG = $(IMAGE_TAG_BASE):$(DEV_VERSION)
 NAMESPACE ?= hc4ai-operator
 
-CONTAINER_TOOL := $(shell command -v docker >/dev/null 2>&1 && echo docker || command -v podman >/dev/null 2>&1 && echo podman || echo "")
+# CONTAINER_TOOL := $(shell command -v docker >/dev/null 2>&1 && echo docker || command -v podman >/dev/null 2>&1 && echo podman || echo "")
 BUILDER := $(shell command -v buildah >/dev/null 2>&1 && echo buildah || echo $(CONTAINER_TOOL))
 PLATFORMS ?= linux/amd64 # linux/arm64 # linux/s390x,linux/ppc64le
 

